recipe/meta.yaml

```diff
@@ -10,25 +10,27 @@ source:
   git_url:  https://github.com/pyg-team/pyg-lib/
   git_rev: {{ version }}
 
+  patches:
+    - fix-cmake.patch
+
 build:
-  number: 1
+  number: 10
   string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
   string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version == "None"]
   # Only build for CUDA >=11.2
   skip: true  # [win or cuda_compiler_version in ("10.2", "11.0", "11.1")]
-
+  
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }}                 # [cuda_compiler_version not in (undefined, 'None')]
-    - sysroot_linux-64 ==2.17                # [linux64]
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - pytorch                                # [build_platform != target_platform]
-    - pytorch =*={{ torch_proc_type }}*      # [build_platform != target_platform]
+    - {{ compiler('cuda') }}                             # [cuda_compiler_version not in (undefined, 'None')]
+    - sysroot_linux-64 ==2.17                            # [linux64]
+    - python                                             # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}                 # [build_platform != target_platform]
+    - pytorch ={{ pytorch }}={{ torch_proc_type }}*      # [build_platform != target_platform]
```

Comment:
I still doubt that we really need `pytorch` in the build environment. It shouldn't be necessary, and actually it risks messing up things.

Reply:
Only the osx-arm64 builds fail without it:

```
  error: dlopen(/Users/runner/miniforge3/conda-bld/pyg-lib_1701263961451/_build_env/venv/lib/python3.8/site-packages/torch/lib/libtorch_global_deps.dylib, 10): image not found
```

See https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=832890&view=logs&j=c35410bf-a820-56fd-e231-d3cd4bead63c&t=b92b255f-9dfe-5c14-4730-460f7a671001

---

I am going to enable it only for osx-arm64. Let me know if you another idea.